sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -279,7 +280,8 @@ public MetricExporter buildMetricExporter() {
      * environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
      */
     public LogRecordExporter buildLogRecordExporter() {
-        internalBuildAndFreeze();
+        ConfigProperties defaultConfig = DefaultConfigProperties.create(Collections.emptyMap());
+        internalBuildAndFreeze(defaultConfig);
         // TODO (trask) how to pass along configuration properties?
         return buildLogRecordExporter(DefaultConfigProperties.create(Collections.emptyMap()));
```

Comment:
(doesn't have to be done in this PR)
```suggestion
        return buildLogRecordExporter(defaultConfig);
```